imgui-SFML.cpp

```diff
@@ -116,6 +117,7 @@ namespace {
 ImColor toImColor(sf::Color c);
 ImVec2 getTopLeftAbsolute(const sf::FloatRect& rect);
 ImVec2 getDownRightAbsolute(const sf::FloatRect& rect);
+void toImVec2Quad(const sf::FloatRect& rect, const sf::Transform& transform, ImVec2 quad[4]);
```

Comment:
This function should return quad instead of accepting it as a parameter

Reply:
If you tell me how to return a c array from a function, without allocating heap mem or declaring it static, I'll do that.